fastapi_rss/models/feed.py

```diff
@@ -107,10 +107,15 @@ def generate_tree(root: etree.Element, dict_: dict):
                     break
             else:
                 RSSFeed._generate_tree_default(root, key, value)
+                if key == "docs" and 'http://www.w3.org/2005/Atom' in nsmap.values():
+                    atom_link = etree.SubElement(root, '{http://www.w3.org/2005/Atom}link', nsmap=nsmap)
+                    atom_link.set('href', value)
+                    atom_link.set('rel', 'self')
+                    atom_link.set('type', 'application/rss+xml')
 
     def tostring(self, nsmap: Optional[Dict[str, str]] = None):
-        nsmap = nsmap or {}
+        nsmap = nsmap or {'atom': 'http://www.w3.org/2005/Atom'}
```

Comment:
Do not default to atom mode. The library must be kept consistent as is (i.e. compliant with the original RSS specification).

In order to ease the implementation I am not opposed to dropping the nsmap kwarg and adding an `atom` / `podcast` kwarg to generate the nsmap accordingly.